hypershift-operator/controllers/nodepool/kubevirt.go

```diff
@@ -58,8 +53,8 @@ func (r NodePoolReconciler) reconcileKubevirtMachineTemplate(ctx context.Context
 	return targetKubevirtMachineTemplate, nil
 }
 
-func kubevirtMachineTemplate(infraName string, hostedCluster *hyperv1.HostedCluster, nodePool *hyperv1.NodePool, controlPlaneNamespace string) (*capikubevirt.KubevirtMachineTemplate, string) {
-	guestQuantity := apiresource.MustParse("4Gi")
+func kubevirtMachineTemplate(nodePool *hyperv1.NodePool, controlPlaneNamespace string) (*capikubevirt.KubevirtMachineTemplate, string) {
```

Comment:
I appreciate you breaking down changes into atomic commits, however this is modifying your previous commit and so tracking local development which has no value in the main history, could you please conflate this changes with the commit that first introduce this i.e https://github.com/openshift/hypershift/pull/779/commits/ab268de78721828c3123857907983264e87c63da

Reply:
I changed the commits to prevent tracking the PR local changes